iOS/DuckDuckGo/SwipeTabsCoordinator.swift

```diff
@@ -133,6 +133,10 @@ class SwipeTabsCoordinator: NSObject {
         }
         
         let indexPath = IndexPath(row: self.tabsModel.currentIndex, section: 0)
+        guard indexPath.row < collectionView.numberOfItems(inSection: 0) else {
```

Comment:
I assume that `scrollToItem` uses the `numberOfItems` function as well so compare against whatever state it thinks it has.